src/backend/base/langflow/graph/vertex/types.py

```diff
@@ -366,6 +366,7 @@ async def _run(self, *args, **kwargs) -> None:
             self._built_result = self._built_object
 
         else:
+            # TODO: _run doesn't exist in ComponentVertex
             await super()._run(*args, **kwargs)
```

Comment:
`ComponentVertex` doesn't have a `_run` method.
Not sure what to do here: ignore or raise exception ?

Reply:
Not sure how to test this either as `self.is_interface_component` is always true for InterfaceVertex.
So this looks like dead code.
Maybe we can just remove the if/else ?

Reply:
I removed the if/else. PTAL.